src/PoojaG/Assignment3/AverageOfArray.java

```diff
@@ -0,0 +1,31 @@
+package PoojaG.Assignment3;
+
+import java.util.Scanner;
+
+public class AverageOfArray {
+	
+	static int averageOfArray(int a[]){
+		int result=0;
```

Comment:
This not necessary every time you get Integer value better to write "double" instead of "int"